File: Assignment1-c/q2.c
//factorial n! say for n=10 or 15, check and trap negative integers
#include<stdio.h>

int main(){
	int n;long long int mul=1;
	printf("Give the integer.\n");
	scanf("%d",&n);
	if(n<0) printf("Give the non-negative integer.\n");
	else{
		int i=1;
		while(i<=n){
			mul*=i;
			i++;
		}
		printf("%d! = %lld \n",n,mul);
	}
	return 0;
}

File: Assignment1/q2.py
# factorial n! say for n=10 or 15, check and trap negative integers
n=int(input("Give the integer "))
mul=1
if n<0:
    print("Give non-negative integer.")
else:
    i=1
    while i<=n:
        mul*=i
        i+=1

    print(f"{n}!= {mul}")